goal_src/jak2/engine/debug/editable-player.gc

```diff
@@ -15,9 +15,6 @@
 
 ;; DECOMP BEGINS
 
-(format 0 "SKIP: editable-player setup~%")
-(format #t "SKIP: editable-player setup~%")
```

Comment:
Does the game start with this removed?

Reply:
I wasn't sure where these came from, looks like the entire file was commented out?  Should probably just be cleared completely if it's causing an issue, we already have enough to keep track of